packages/frontend2/src/app/layout.tsx

```diff
@@ -58,7 +58,11 @@ export default async function RootLayout({
           enabled={env.NEXT_PUBLIC_PLAUSIBLE_ENABLED}
         >
           <TRPCReactProvider>
-            <ThemeProvider attribute="class" storageKey="l2beat-theme">
+            <ThemeProvider
+              attribute="class"
+              storageKey="l2beat-theme"
+              disableTransitionOnChange
```

Comment:
I guess if we add this here then we want to remove the transition-avoidance stuff from the navbar